docs/src/neural_ode/neural_gde.md

```diff
@@ -149,94 +266,75 @@ model = Flux.Chain(GCNConv(adj_mat, num_features=>hidden, relu),
 
 We shall be using the standard categorical crossentropy loss function which is used for multiclass classification tasks.
 
-```julia
-loss(x, y) = logitcrossentropy(model(x), y)
-accuracy(x, y) = mean(onecold(model(x)) .== onecold(y))
-```
-
-### Model Parameters
+```@example graphneuralode
+logitcrossentropy(ŷ, y) = mean(-sum(y .* logsoftmax(ŷ); dims=1))
 
-Now we extract the model parameters which we want to learn.
+function loss(x, y, mask, model, ps, st)
+    ŷ, st = model(x, ps, st)
+    return logitcrossentropy(ŷ[:,mask], y), st
+end
 
-```julia
-ps = Flux.params(model, node.p);
+function eval_loss_accuracy(X, y, mask, model, ps, st)
+    ŷ, _ = model(X, ps, st)
+    l = logitcrossentropy(ŷ[:,mask], y[:,mask])
+    acc = mean(onecold(ŷ[:,mask]) .== onecold(y[:,mask]))
+    return (loss = round(l, digits=4), acc = round(acc*100, digits=2))
+end
 ```
 
-### Training Data
-
-GNNs operate on an entire graph, so we can't do any sort of minibatching here. We need to pass the entire data in a single pass. So our dataset is an array with a single tuple.
+### Setup Model
+We need to manually set up our mode with `Lux`, and convert the paramters to `ComponentArray` so that they can work well with sensitivity algorithms.
+```@example graphneuralode
+rng = Random.default_rng()
+Random.seed!(rng, 0)
 
-```julia
-train_data = [(train_X, train_y)]
+ps, st = Lux.setup(rng, model)
+ps = ComponentArray(ps) |> device
+st = st |> device
 ```
-
 ### Optimizer
 
 For this task we will be using the `ADAM` optimizer with a learning rate of `0.01`.
 
-```julia
-opt = ADAM(0.01)
-```
-
-### Callback Function
-
-We also define a utility function for printing the accuracy of the model over time.
-
-```julia
-evalcb() = @show(accuracy(train_X, train_y))
+```@example graphneuralode
+opt = Optimisers.ADAM(0.01f0)
```

Comment:
```suggestion
opt = Optimisers.Adam(0.01f0)
```

Reply:
Optimisers.jl changed the capitalizations from the Flux convention.

Reply:
Good to know that